packages/gatsby-theme-apollo-docs/src/components/page-content.js

```diff
@@ -221,6 +248,11 @@ export default function PageContent(props) {
             imagesLoaded={imagesLoaded === imagesToLoad}
           />
         )}
+        { props.ffWidgetId && (
+          <FeedbackLink>
+            <IconStar style={starIconOffset} /> Rate article
```

Comment:
You could put this line inside the `FeedbackLink` component, render `<FeedbackLink />` here, and avoid spreading `...props` inside that component.

You could also one-line the `starIconOffset` to just `marginTop: -2`:

```jsx
<IconStar style={{marginTop: -2}} />
```